library/core/src/ptr/mut_ptr.rs

```diff
@@ -2197,3 +2213,54 @@ impl<T: ?Sized> PartialOrd for *mut T {
         *self >= *other
     }
 }
+
+#[cfg(kani)]
+#[unstable(feature = "kani", issue = "none")]
+mod verify {
+    use crate::kani;
+
+    macro_rules! generate_mut_add_and_sub_harness {
+        ($type:ty, $proof_name:ident, $func_name:ident) => {
+            #[allow(unused)]
```

Comment:
this line could be a put outside of macro as general case?

Reply:
ok let me check if that works.